statistics.py

```diff
@@ -146,3 +161,53 @@ def print_most_often_replies_to(chat: Chat) -> None:
                 f"{replied_sender.name:20} was most often replied by",
                 ", ".join([f"{entry[0].name} ({entry[1]} times)" for entry in top_replies])
             )
+
+
+def _parse_string_to_words(text: str) -> List[str]:
+    matches = WORD_REGEX.findall(text)
+    words = []
+    for match in matches:
+        words.append(match[0])
+    return words
```

Comment:
Смотри как можно. И быстрее и красивее)
```suggestion
    return [match[0] for match in WORD_REGEX.findall(text)]
```